robosuite/scripts/collect_human_demonstrations.py

```diff
@@ -82,9 +167,29 @@ def collect_human_trajectory(env, device, arm, max_fr):
                 action_dict[arm] = input_ac_dict[f"{arm}_abs"]
             else:
                 raise ValueError
+        all_robot_action_dicts = []
+        # set inactive robot action dicts
+        for i, robot in enumerate(env.robots):
+            if i == device.active_robot:
+                all_robot_action_dicts.append(action_dict)
+                continue
+
+            inactive_robot_ac_dict = copy.deepcopy(all_prev_gripper_actions[i])
```

Comment:
Setting inactive robot action as prev gripper action is confusing to me, as in active robot action seems more than just gripper. Can we rename things to be more intuitive?